index.js

```diff
@@ -22,7 +23,13 @@ class RPCClient {
         args: JSON.stringify(args),
       }),
       credentials: this.sendCredentials,
-    })
+    }
+    if (process.env.NODE_ENV !== 'production') {
+      options.agent = new https.Agent({
+        rejectUnauthorized: false
+      });
+    }
```

Comment:
Regarding this change, having such a low-lever library be aware of the environment feels a bit off to me. I would rather expose some options of the underlying library to the consuming code, but this would be my second option.

I would even consider another library that has better configuration than fetch... Our apps are big enough that having a more feature-packed library like `asxios` may not be an issue at all.